addon/components/utils/address-form.ts

```diff
@@ -30,13 +29,52 @@ export default class extends Component<UtilsAddressFormArgs> {
   validPhoneNumber: boolean = true;
   countries = countries;
 
+  get useGoogleAutocomplete(): boolean {
+    return this.args.useGoogleAutocomplete ?? true;
+  }
+
+  @action
+  onSetup() {
+    if (isTesting()) return;
```

Comment:
![hello](https://github.com/upfluence/ember-upf-utils/assets/5032005/38c3f4aa-b830-4de8-9240-dffde95969a8)

Reply:
I keep it for test, we don't want to setup the google autocomplete for that